lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -952,6 +953,17 @@ def _create_deployment_parameters(
             parameters=parameters,
         )
 
+        # dump arm_template and arm_parameters to file
+        dump_path = constants.RUN_LOCAL_PATH / "arm_dump"
+        try:
+            os.mkdir(dump_path)
```

Comment:
The log_path of environment will be created automatically, when it's accessed. Check the code logic.